src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductVariantRepository.php

```diff
@@ -162,6 +162,9 @@ public function findByPhrase(string $phrase, string $locale): array
             ))
             ->setParameter('phrase', '%' . $phrase . '%')
             ->setParameter('locale', $locale)
+            ->orderBy('o.product', 'ASC')
+            ->addOrderBy('o.position', 'ASC')
+            ->setMaxResults($limit === 0 ? null : $limit)
```

Comment:
Is this a side effect of the type casting `'!!int $limit'`?

Reply:
Yes. `0` is unwanted behavior. I would like nullable int instead of forced/casted int.